packages/nextjs-mf/src/plugins/container/runtimePlugin.ts

```diff
@@ -172,7 +178,8 @@ export default function (): FederationRuntimePlugin {
         return args;
       }
       args.resolver = function () {
-        shareScopeMap[scope][pkgName][version] = host.options.shared[pkgName]; // replace local share scope manually with desired module
+        shareScopeMap[scope][pkgName][version] =
+          host.options.shared[pkgName][0]; // replace local share scope manually with desired module
```

Comment:
@ScriptedAlchemy  Because instances.options.shared is array, so i use the first item directly . Will users set muliple version react ?

Reply:
Probably not, but user may need to if they make runtime plugin like automatic adapter for multiple framework or multi react. Like if remote is react 15, pick react 15 from array list and then for onLoad, mount new app onto DIV in adapter. 

For next, it will always just get 1 version of react, the hosts